sig-testing/flakiness-sla.md

```diff
@@ -0,0 +1,40 @@
+# Kubernetes Testing Flakiness SLA
+
+This document captures the expectations of the community about flakiness in
+our tests and our test infrastructure. It sets out an SLA (Service Level 
+Agreement) for flakiness in our tests, as well as actions that we will
+take when we are out of SLA.
+
+
+## Definition of a "Flake"
+
+We'll start by the definition of a _flake_. A test result is considered to
```

Comment:
Do we differentiate flake by environment today?  For example is a flake on kops via aws different than a flake on GCE or GCI?  Is a node e2e w or w/o CRI enabled a different flake?  Basically i wonder what makes a flake unique?  Is it test plus environment or just test?